### PR TITLE
Create Fetch_DT.py

### PR DESCRIPTION
**Pull request by devdhruvper**
Code to fetch data for any given year by - Dhruv Talan

### CODE REVIEW

**Comment by SriPrarabdha** [comment]
@devdhruvper  Rather that printing out the results store them in a array of dicts . The final array 'data' should have all the data in following form
`
data = [
                {
                     'id' : '567549' ,
                     'date' : 'month+Year' ,
                     'tagline' : "Heading of that judgment" ,
                      'judgment' : "Actual Judgment .... " ,
               },
                {
                     'id' : '567548' ,
                     'date' : 'month+Year' ,
                     'tagline' : " Heading of that judgment" ,
                      'judgment' : "Actual Judgment .... ",
               },
           ]
`

**Comment by SriPrarabdha** [comment]
You will get id in url ,for date use 'datetime' module in python

**Comment by devdhruvper** [comment]
Changed into required JSON format

**Comment by SriPrarabdha** [comment]
@devdhruvper There seems to be issue with the date as you can see in the ss below date for a judgment on December 1950 is tagged as 16/1950
Try to debug this!!
json

File: Fetch_DT.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+     Link_container.append(links)
+     numbers.append(number)
+
+   for URL, no in zip(Link_container, numbers):
+    R = requests.get(URL)
+    HtmlContent = R.content
+    soup = BeautifulSoup(HtmlContent, 'html.parser')
+    page_div = soup.find("div", class_="judgments")
+    tagline = page_div.find("div", class_="doc_title").text
+    ad_doc = page_div.find("div", class_="ad_doc")
```
Comment (SriPrarabdha) [inline Fetch_DT.py]: remove this ad_doc instead add
try:
     judgment = page_div.find("pre").text
    except:
     judgment = ""
     p_tags = page_div.find_all("p")
     for p_tag in p_tags:
      judgment += p_tag.text
data.append({'id': no, 'date': f"{i}/{Year}", 'tagline': tagline , 'judgment':judgment})

**Review by SriPrarabdha — CHANGES_REQUESTED** [review]

**Comment by devdhruvper** [comment]
solved the bug in months and added judgment.

**Pull request merged**